### DISCUSSION TITLE
Linux executable of Standalone Faster-Whisper

### DISCUSSION THREAD

**Discussion by Purfview** [Purfview/whisper-standalone-win#84] [Announcements]
Released Linux executable of Standalone Faster-Whisper.

Earlier versions and r192.3.1 - Should be compatible with Linux kernel v5.4 and above.

r245.2 - Should be compatible with Linux kernel v5.15 and above.

**Comment by rodnvs** [comment]
Hello @Purfview ! I appreciate a lot this project, and the improvements that you made upon Faster-Whisper! Is it any plan to release a linux build for the r192.3.4 version? Thx

**Reply by Purfview** [reply]
https://github.com/Purfview/whisper-standalone-win/discussions/332
